packages/@apphosting/adapter-angular/src/bin/build.ts

```diff
@@ -1,20 +1,16 @@
 #! /usr/bin/env node
-import { spawn } from "child_process";
-import { loadConfig } from "../utils.js";
+import {
+  loadConfig,
+  build,
+  populateOutputBundleOptions,
+  generateOutputDirectory,
+} from "../utils.js";
 
-const build = (cwd = process.cwd()) =>
-  new Promise<void>((resolve, reject) => {
-    // TODO warn if the build script contains anything other than `ng build`
-    const process = spawn("npm", ["run", "build"], { cwd, shell: true, stdio: "inherit" });
-    process.on("exit", (code) => {
-      if (code === 0) return resolve();
-      reject();
-    });
-  });
+const cwd = process.cwd();
 
-const config = await loadConfig(process.cwd());
+// read Application Builder config
```

Comment:
probably better to move this comment to describe the function

Reply:
done